<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QMainWindow" name="MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainPage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="holder">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1536</width>
      <height>864</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="mainWindow">
     <widget class="QFrame" name="frame1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1536</width>
        <height>864</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frame1 {
	background-image: url(:/bg/mainpage.png);
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
    border: none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QPushButton" name="getStarted">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>510</y>
         <width>291</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Book Antiqua</family>
         <pointsize>22</pointsize>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#getStarted {
    background-color: #DC3E22;
    color: white;
    border-radius: 10px; 
    font-weight: bold;
}
QPushButton#getStarted:hover {
    background-color: #c0392b;  /* Darker red on hover */
}
QPushButton#getStarted:pressed {
    background-color: #a93226;  /* Even darker red when pressed */
}</string>
       </property>
       <property name="text">
        <string>Get Started</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="loginPage">
     <widget class="QFrame" name="frame2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1536</width>
        <height>864</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frame2 {
	background-image: url(:/bg/loginn.png);
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
    border: none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QPushButton" name="login">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>430</y>
         <width>151</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Book Antiqua</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
   background-color: #DC3E22;
    color: white;
    border-radius: 10px; 
    font-weight: bold;
}
QPushButton:hover {
    background: #b33b1b; 
}
QPushButton:pressed {
    background: #992f15; 
}</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>250</y>
         <width>341</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid #DC3E22; 
    border-radius: 5px;        
    background: transparent;  
    padding: 8px;
	color: rgb(255, 255, 255);
    font-siza: 18px
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="password">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>340</y>
         <width>341</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid #DC3E22; 
    border-radius: 5px;        
    background: transparent;  
    padding: 8px;
	color: rgb(255, 255, 255);
    font-siza: 18px
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>230</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>320</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>580</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Book Antiqua</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: transparent;          
    border: 2px solid #DC3E22;         
    border-radius: 5px;                
    color: white;                       
    padding: 5px 10px;                 
}
QPushButton:hover {
    background: rgba(220, 62, 34, 0.1);  
}
QPushButton:pressed {
    background: rgba(220, 62, 34, 0.2); 
}
</string>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="customerPage">
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1536</width>
        <height>864</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frame_2 {	
	background-image: url(:/bg/fitora.png);
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
    border: none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QPushButton" name="logOut_4">
       <property name="geometry">
        <rect>
         <x>1390</x>
         <y>20</y>
         <width>83</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Book Antiqua</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: transparent;          
    border: 2px solid #DC3E22;         
    border-radius: 5px;                
    color: white;                       
    padding: 5px 10px;                 
}
QPushButton:hover {
    background: rgba(220, 62, 34, 0.1);  
}
QPushButton:pressed {
    background: rgba(220, 62, 34, 0.2); 
}</string>
       </property>
       <property name="text">
        <string>logout</string>
       </property>
      </widget>
      <widget class="QPushButton" name="searchButton">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>140</y>
         <width>561</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: transparent;          
    border: 2px solid #DC3E22;         
    border-radius: 20px;                
    color: white;                       
    padding: 5px 10px; 
   
}
QPushButton:hover {
    background: rgba(220, 62, 34, 0.1);  
}
QPushButton:pressed {
    background: rgba(220, 62, 34, 0.2); 
}</string>
       </property>
       <property name="text">
        <string>search for available courts</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="search">
     <widget class="QCommandLinkButton" name="goHomeButton1">
      <property name="geometry">
       <rect>
        <x>1240</x>
        <y>10</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::GoHome"/>
      </property>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1536</width>
        <height>864</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frame_6 {	
	background-image: url(:/bg/fitora.png);
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
    border: none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QTableWidget" name="courtTable">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>80</y>
         <width>891</width>
         <height>421</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="search_2">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>520</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>search</string>
       </property>
      </widget>
     </widget>
     <zorder>frame_6</zorder>
     <zorder>goHomeButton1</zorder>
    </widget>
    <widget class="QWidget" name="coachPage">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>70</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>coach</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logOut_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>240</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>logout</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1536</width>
        <height>864</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frame {	
	background-image: url(:/bg/fitora.png);
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
    border: none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
     <zorder>frame</zorder>
     <zorder>label_3</zorder>
     <zorder>logOut_3</zorder>
    </widget>
    <widget class="QWidget" name="receptionistPage">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>170</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>receptionist</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1536</width>
        <height>864</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frame_4 {	
	background-image: url(:/bg/fitora.png);
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
    border: none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QPushButton" name="clientData">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>320</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Client Data</string>
       </property>
      </widget>
      <widget class="QPushButton" name="logOut">
       <property name="geometry">
        <rect>
         <x>1290</x>
         <y>50</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>logout</string>
       </property>
      </widget>
     </widget>
     <zorder>frame_4</zorder>
     <zorder>label_6</zorder>
    </widget>
    <widget class="QWidget" name="managerPage">
     <widget class="QPushButton" name="logOut_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>270</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>logout</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>100</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>manager</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1536</width>
        <height>864</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frame_3 {	
	background-image: url(:/bg/fitora.png);
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
    border: none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
     <zorder>frame_3</zorder>
     <zorder>logOut_2</zorder>
     <zorder>label_5</zorder>
    </widget>
    <widget class="QWidget" name="clientDataPage">
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1536</width>
        <height>864</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frame_5 {	
	background-image: url(:/bg/fitora.png);
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
    border: none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QPushButton" name="getClientData">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>250</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Get Data</string>
       </property>
      </widget>
      <widget class="QPushButton" name="backToRes">
       <property name="geometry">
        <rect>
         <x>1260</x>
         <y>50</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="clientID">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>190</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>190</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>330</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>370</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Date of Birth</string>
       </property>
      </widget>
      <widget class="QLabel" name="getClientName">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>330</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="getClientDateOfBirth">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>370</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>420</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Subscription Period</string>
       </property>
      </widget>
      <widget class="QLabel" name="getSubscriptionPeriod">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>420</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
